sdcm/sct_events/grafana.py

```diff
@@ -112,13 +111,17 @@ def run(self) -> None:
 
         for annotation in self.inbound_events():  # events from GrafanaAggregator
             with verbose_suppress("GrafanaEventPostman failed to post an annotation %s", annotation):
-                requests.post(self._grafana_post_url, json=annotation, auth=self.api_auth).raise_for_status()
+                for grafana_post_url in self._grafana_post_urls:
+                    requests.post(
+                        grafana_post_url, json=annotation, auth=self.api_auth).raise_for_status()
```

Comment:
raising while in a loop, would mean it won't be sent to the next urls

Reply:
Fixed.